docs/concepts/alert_activations.ipynb

```diff
@@ -0,0 +1,552 @@
+     "output_type": "stream",
+     "text": [
+      "> 2024-12-11 11:52:10,062 [info] Project loaded successfully: {\"project_name\":\"default\"}\n"
+     ]
+    }
+   ],
+   "source": [
+    "project = mlrun.get_or_create_project(\"default\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "a012c3d0-11ce-41f1-bc9b-e8fb8d4967b3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# to get activations for specific project\n",
+    "activations = project.list_alert_activations(\n",
+    "    name=None,\n",
+    "    since=None,\n",
+    "    until=None,\n",
+    "    entity=None,\n",
+    "    severity=None,\n",
+    "    entity_kind=None,\n",
+    "    event_kind=None,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "5ae0441c-5105-47c3-aa43-78ce8215f8fd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# to get activations for specific alert config\n",
+    "alert_config = run_db.list_alerts_configs()[0]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "842fa453-a434-4794-a1a4-149b65d900d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "activations = alert_config.list_activations(\n",
+    "    since=None,\n",
+    "    until=None,\n",
+    "    from_last_update=False,  # set to false to get activations only from the time when alert config was updated (takes precedence over since of both are passed)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0fd1abc-9f92-46eb-b787-19721b79c323",
+   "metadata": {},
+   "source": [
+    "## Filter by entity\n",
+    "The `entity_id` for the JOB `entity_kind` is formatted as `<job-name>.<job_uid>`. This is the only field in alert activation that supports wildcard search. To enable wildcard search, the entity parameter should begin with a `~`.\n",
+    "\n",
+    "For example, if you know the job name and want to find all activations related to it, you can pass the entity parameter as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "e2961fcd-ccf8-4f2e-a944-08baa86a0ad9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "test-func-handler.db80cba0c4be4ee9b86a09cf12a89991\n",
+      "test-func-handler.db80cba0c4be4ee9b86a09cf12a89991\n",
+      "test-func-handler.58e25426ea154daab2afa4ebfe454c71\n",
+      "test-func-handler.58e25426ea154daab2afa4ebfe454c71\n"
+     ]
+    }
+   ],
+   "source": [
+    "activations = project.list_alert_activations(entity=\"~test-func-handler.*\")\n",
+    "for activation in activations:\n",
+    "    print(activation.entity_id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "236497c2-e1eb-4f34-a8b5-a18c4f9969d7",
+   "metadata": {},
+   "source": [
+    "If the `entity` parameter is passed as a string without a tilde (~), the search will be performed for an exact match with the given string:"
```

Comment:
"If the `entity` parameter is passed as a string without a tilde (~), the search is performed for an exact match with the given string:"